src/signals/incident/components/form/MapSelectors/Asset/Selector/NearbyLayer/NearbyLayer.tsx

```diff
@@ -148,6 +161,9 @@ export const NearbyLayer: FC<NearbyLayerProps> = ({ zoomLevel }) => {
 
     data.features.forEach((feature) => {
       const { lat, lng } = featureToCoordinates(feature.geometry)
+
+      if (findAssetMatch(assetData, lat, lng)) return
```

Comment:
maybe a small description

Reply:
I've added a description and `return null`